src/MinFraud/ReportTransaction.php

```diff
@@ -59,22 +61,25 @@ public function __construct(
      *                               a minFraud Standard or Premium request. These IDs are
      *                               returned in the maxmindID field of a response for a
      *                               successful minFraud request. This field is not
-     *                               required, but you are encouraged to provide it, if
-     *                               possible.
+     *                               required if you provide at least one of the transaction's
+     *                               ipAddress, minfraudId, or transactionId. You are
+     *                               encouraged to provide it, if possible.
      * @param string $minfraudId     Optional. A UUID that identifies a minFraud Score,
      *                               minFraud Insights, or minFraud Factors request. This
      *                               ID is returned at /id in the response. This field is
-     *                               not required, but you are encouraged to provide it if
-     *                               the request was made to one of these services.
+     *                               not required if you provide at least one of the transaction's
+     *                               ipAddress, maxmindId, or transactionId. You are encouraged to
+     *                               provide it the request was made to one of these services.
      * @param string $notes          Optional. Your notes on the fraud tag associated with
      *                               the transaction. We manually review many reported
      *                               transactions to improve our scoring for you so any
      *                               additional details to help us understand context are
      *                               helpful.
      * @param string $transactionId  Optional. The transaction ID you originally passed to
-     *                               minFraud. This field is not required, but you are
-     *                               encouraged to provide it or the transaction's
-     *                               maxmind_id or minfraud_id.
+     *                               minFraud. This field is not required if you provide at
+     *                               least one of the transaction's ipAddress, maxmindId, or
+     *                               minfraudId. You are encouraged to provide it or the
+     *                               transaction's maxmind_id or minfraud_id.
```

Comment:
```suggestion
     *                               transaction's maxmindId or minfraudId.
```

I suppose we could put backticks around all of these, but that is preexisting.